docs/sources/_index.md

```diff
@@ -54,30 +58,18 @@ hero:
 
 Collecting the relevant telemetry data, such as metrics, logs, and traces, for analysis is an indispensable part of understanding the health of your system.
 
-{{< param "PRODUCT_NAME" >}} is a multifunction tool you can use to collect, process, and forward telemetry data to the observability backend of your choosing.
+{{< param "PRODUCT_NAME" >}} is more than just a collector. You can use {{< param "PRODUCT_NAME" >}} to collect, process, and forward telemetry data to the observability backend of your choosing.
 
 {{< param "PRODUCT_NAME" >}} has the following features to help you customize, scale, secure, and troubleshoot your data pipeline.
-1. Custom components
-1. GitOps compatibility
-1. Clustering support
-1. Security
-1. Debugging utilities
 
-Check out the {{< param "PRODUCT_NAME" >}} [Introduction] page for more information on these and other key features.
+* Custom components
+* GitOps compatibility
+* Clustering support
+* Security
+* Debugging utilities
 
 {{< param "PRODUCT_NAME" >}} is flexible, and you can easily configure it to fit your needs in on-prem, cloud-only, or a mix of both.
 
```

Comment:
One thing to add here is a clear call to action, which is for them to install, configure, and collect forward telemetry data. 

We could do that by including the section on getting started in the original draft on the overview page

OR 

include a blurb saying, "Getting started with Alloy consists of 3 major steps.
Click here to get started." Then, link it to a new page with content in the original draft as shown below.

New page: 
Getting started with Alloy consists of 3 major steps:
1. Install {{< param "PRODUCT_NAME" >}} 
1. Configure {{< param "PRODUCT_NAME" >}} 
1. Collect and forward telemetry data to the observability backend of your choice.

In addition, you can use the Grafana dashboard to visualize the data collected from the app or infrastructure.

For a quick overview of this process, check out the following tutorials.
* [Use Grafana Alloy to send logs to Loki](https://grafana.com/docs/alloy/latest/tutorials/send-logs-to-loki/)
* [Use Grafana Alloy to send metrics to Prometheus](https://grafana.com/docs/alloy/latest/tutorials/send-metrics-to-prometheus/)
* For more tutorials, check out the [tutorial page](https://grafana.com/docs/alloy/latest/tutorials/)

What do you think?

Reply:
Some thougths on this:

* I'd really like to keep this landing page as simple as possible... and in-line with the other Grafana product landing pages.
* We should never use `Click here` in any documentation.
* The `3 major steps` part doesn't fit in so well... if we want to dive into this level of detail, then it should be on a child page.
* The landing page should simply introduce the product, not provide steps/instructions.

Reply:
Agreed. Why don't we leave out the call to action part and include the suggested text to the Get started page?
https://grafana.com/docs/alloy/latest/get-started/